spec/Common.hs

```diff
@@ -1,8 +1,16 @@
-module Common where
+module Common
+  ( shouldRenderFrom
```

Comment:
I think there is no need to filter the exports. Or was there a name clash?

Reply:
No name clash. I introduced a new top level binding of `testRender` & was preventing its export. It's not a big deal either way.

Reply:
Ah, I'm fine if you don't restrict the exports in that case.